app/src/main/java/com/rob729/quiethours/Fragments/NewProfileFragment.kt

```diff
@@ -187,6 +213,22 @@ class NewProfileFragment : Fragment() {
             "$i"
         }
     }
+    private fun dayPicker(d: List<Boolean>, materialDayPicker: MaterialDayPicker) {
```

Comment:
The same function is being used at two places so create a Utils,kt object type file and place this function there and access it from there only.